### PR TITLE
adding pt-br translation

### PR DESCRIPTION
**Pull request by Pompeu**

### CODE REVIEW

**Comment by ddnexus** [comment]
Thanks for this. Do you think it could be possible to get a single translation that could work for both pt-br and for pt-pt?

**Comment by Pompeu** [comment]
> Thanks for this. Do you think it could be possible to get a single translation that could work for both pt-br and for pt-pt?

i dont know!
same libs use pt-br  and only pt for translate, like momentjs, maybe we can use onlye pt ?
if u like i change for pt only.

**Comment by ddnexus** [comment]
@Pompeu: According to @gamafranco something of what is in the pt-br would not sound right in pt-pt. My request was if you could find a common way to translate it, so it would work for pt-br and pt-pt. @gamafranco ?

File: lib/locales/pagy.yml [hunk 1]
```diff
@@ -46,6 +46,31 @@ es:
...
+      prev: "&lsaquo;&nbsp;Anterior"
+      next: "Próximo&nbsp;&rsaquo;"
+      gap: "&hellip;"
+      current: "Esta na página"
+    info:
+      single_page:
+        zero: "Sem resultados"
+        one: "Mostrando 1 %{item_name}"
+        other: "Mostrando %{count} %{item_name}"
+      multiple_pages: "Mostrando %{item_name} %{from}-%{to} de %{count} no total"
```
[the remark this answers is not in the document]
Comment (Pompeu) [reply lib/locales/pagy.yml]: > "Mostrando %{from}-%{to} %{item_name} de %{count} no total"

myabe ? Mostrando %{item_name} %{from}-%{to} no total de %{count}

**Comment by Pompeu** [comment]
> @Pompeu: According to @gamafranco something of what is in the pt-br would not sound right in pt-pt. My request was if you could find a common way to translate it, so it would work for pt-br and pt-pt. @gamafranco ?

i dont know, i see all time in same libs, pt-br and pt-pt, i see this in subtitles of films too.

**Comment by ddnexus** [comment]
@Pompeu: yes, that is what I mean... however IIRC, in a private chat @gamafranco told me something about "Mostrar" that is not used in regular Portuguese. So what I was asking is translating the file in such a way that it would sound right for both Brazilian and Portuguese.

**Comment by gamafranco** [comment]
The problem is not with "mostrar", it's with "mostrando"

**Comment by gamafranco** [comment]
We use "a mostrar" instead of "mostrando"

**Comment by ddnexus** [comment]
@Pompeu, @juuh42dias Would "a mostrar" work also for pt-br?

**Comment by Pompeu** [comment]
> @Pompeu, @juuh42dias Would "a mostrar" work also for pt-br?
no make sense in pt-br  use of  "a mostar",  why mostrando == showing == "a mostrar",
generally in Brazil we dont use "a mostrar" ( "a"  is not a prefix here, because we use "ando" is same as "ing" in english.

**Comment by ddnexus** [comment]
@Pompeu @juuh42dias I suggest to post a quick round of alternative translation that work for pt-br...
@gamafranco same for a few alternatives for pt-pt...
Then we may find a common translation OR we could add 2 different locales.

**Pull request merged**